examples/produce_part_sweep/README.md

```
@@ -0,0 +1,35 @@
+# /pub/examples/partcad/produce_part_sweep
+
+This example demonstrates how to define 3D parts by sweeping 2D sketches.
+
+## Usage
+```shell
+pc inspect dxf
+pc inspect cylinder
+pc inspect clock
+```
+
+
```

Markdownlint (0.37.0)

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)